benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -121,16 +195,18 @@ def noop():
     model, inp = model_getter(device)
 
     v = get_v_for(model, inp, task)
-    # Warmup
-    run_once(model, inp, task, v)
 
-    elapsed = []
-    for it in range(args.num_iters):
-        do_sync()
-        start = time.time()
-        run_once(model, inp, task, v)
-        do_sync()
-        elapsed.append(time.time() - start)
+    with torch.set_grad_enabled(run_once_fn == run_once):
+        # Warmup
+        run_once_fn(model, inp, task, v, check_consistency=run_once_fn == run_once_functorch)
```

Comment:
Function equality seems risky if someone comes along in the future and adds a decorator to one of these functions. Maybe we should call it `maybe_check_consistency` and have it always be True. ("maybe" bcause `run_once` doesn't check consistency and run_once_functorch does haha)

Reply:
I like `maybe_check_consistency` :)